util/mod_translation_updater.py

```diff
@@ -264,7 +263,7 @@ def read_lua_file_strings(lua_file):
 
 		for s in strings:
 			s = re.sub(r'"\.\.\s+"', "", s)
-			s = re.sub("@[^@=0-9]", "@@", s)
+			s = re.sub("@(?=[^0-9])", "@@", s)
```

Comment:
`@0` is not a valid escape:

https://github.com/minetest/minetest/blob/56902745c8265a8187b2c3855722fa549d1caaf7/doc/lua_api.md?plain=1#L4095-L4096

-------

I also just realised that this substitution does not have any effects in implementation since the original string needs to be correctly escaped in order to work, e.g. `S("@a @b @c @d @e @f")` would **not** work, but `S("@@a @@b @@c @@d @@e @@f")` would.

Reply:
Maybe the script should reject these strings then (and print a warning)